src/exchanges/fetch.ts

```diff
@@ -0,0 +1,16 @@
+import { Exchange } from '../types';
+import { mergeMap } from 'rxjs/operators';
+
+export const fetchExchange = (): Exchange => () => ops$ =>
+  ops$.pipe(
```

Comment:
Quick recap: This can mostly use the old implementation (at least the factory) without any modifications except the error behaviour that's emitting error events. It's important we keep `CombinedError` around.